table_helper.hh

```diff
@@ -95,8 +97,27 @@ public:
         }));
     }
 
+    /**
+     * Execute a single insertion into the table possibly using fallback cql_options.
+     *
+     * @tparam OptMaker cql_options maker functor type
+     * @tparam Args OptMaker arguments' types
+     * @param opt_maker cql_options maker functor
+     * @param opt_maker_fallback cql_options maker functor for the fallback CQL INSERT statement
+     * @param opt_maker_args opt_maker arguments
+     */
```

Comment:
Please add constraints for all template parameters, both to catch errors, and as a more exact form of documentation.

Reply:
Ugh, you have a constraint. But really, I still can't guess what an opt_maker is.